tests/unit/commands/new-test.js

```diff
@@ -90,4 +102,53 @@ describe('new command', function() {
       expect(error.message).to.equal('Trying to generate an application structure in this directory? Use `ember init` instead.');
     });
   });
+
+  it('registers blueprint options in beforeRun', function() {
+    stub(Blueprint, 'lookup', function(name) {
+      expect(name).to.equal('app');
+      return {
+        availableOptions: [
+          { name: 'custom-blueprint-option', type: String }
+        ]
+      };
+    }, true);
+
+    try {
+      command.beforeRun(['app']);
+      expect(pluck(command.availableOptions, 'name')).to.contain('custom-blueprint-option');
+    } finally {
+      safeRestore(Blueprint, 'lookup');
```

Comment:
Okay. I didn't want to stub out ```lookup``` in every test, but you're suggesting that I put the ```safeRestore``` in ```afterEach```, but keep the ```stub``` in the test body?